weaviate/pyproject.toml

```
@@ -0,0 +1,62 @@
+[build-system]
+requires = [
+    "hatchling>=0.13.0",
+    "setuptools>=66; python_version > '3.0'",
+    "setuptools; python_version < '3.0'",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "datadog-weaviate"
+description = "The Weaviate check"
+readme = "README.md"
+license = {text = "BSD-3-Clause"}
+requires-python = ">=3.8"
```

```suggestion
requires-python = ">=3.9"
```

Since we only test with py3.9